src/components/accordion/accordion.tsx

```diff
@@ -34,7 +34,7 @@ export const Accordion: FC<AccordionProps> = ({children, expandMode = 'single'})
     () =>
       _(renderChildrenSpecifiedComponents(children, ['AccordionSection']))
         .compact()
-        .value(),
+        .value() as React.ReactElement[],
```

Comment:
Could we better type `accordionSections` itself to reduce the casting throughout the file?